### ISSUE TITLE
In Sept - Read about iDrive online server backup service, 8-7-17

### ISSUE THREAD

**Issue by david-huddleston** [soccerhelp/soccerhelp#321]
I emailed this on 8-7-17 at 2:47 PM

Dustin, check out below and let's discuss. It only costs $99 a year for 250 GB

for Unlimited Servers, Unlimited Computers and Unlimited Users.

If it has enough room, you could back your stuff up there too.

We should be able to put a back up of the files on both servers and even the Videos there so everything is in one place.

>>> You can Back-Up both servers there as an extra back-up AND also you can back up your personal computer.  <<<<< 

Look at the Business Plan  - they got the highest rating from PC Magazine

It says they only back-up the parts of it that are changed. You might also be able to set up a schedule for back-ups

You can see the features at the link below >

Pricing > https://www.idrive.com/pricing 

Backup Features >   https://www.idrive.com/online-backup-features 

See the review at >

http://www.pcmag.com/article2/0,2817,2288745,00.asp

and  https://www.pcmag.com/article2/0,2817,2362675,00.asp 

PROS
Easy setup. Unlimited devices in one account. Continuous backup option. Disk image backup (the entire image). File Explorer integration. Folder syncing. File archiving. Hard drive for mailing in data.
CONS
Slightly slower performance than some competitors.
BOTTOM LINE
You'd be hard-pressed to find an online backup service as full-featured or versatile as IDrive, especially for the price.

**Comment by dustinschaerer** [comment]
@david-huddleston This seems like a viable option if you want me to set this up.  It has Linux support, but it's not as full featured as the Windows and Mac support.

**Comment by dustinschaerer** [comment]
@david-huddleston Unfortunately the documentation for the iDrive PERL scripts leaves much to be desired. I don't know the PERL language and I apparently have to run everything from the command prompt on the remote server.  It'll take a bit of studying to figure this all out.
